kubernetes/README.md

```diff
@@ -154,5 +154,32 @@ You can use the other Kubernetes run target to build the image and deploy the pr
 
 ## Setup existing Kubernetes projects with the plugin
 
-(coming soon)
-
+You can open any project with a configured container image build (using Dockerfile or [Jib](https://github.com/GoogleContainerTools/jib)) and Kubernetes manifests, and use it with the plugin. The only additional bit of configuration is a Skaffold YAML file that you can create from a provided template. Here is a quick tutorial:
```

Comment:
suggestion:

`You can open any project with a configured container image build (using Dockerfile or Jib) and Kubernetes manifests, and use it with the plugin.`

->

`You can open any existing project already configured with Kubernetes manifests and Dockerfile (or Jib), and use it with the plugin.`